### PR TITLE
Added raise6k dataset

### PR DESCRIPTION
**Pull request by shahrukh-dl**

### CODE REVIEW

**Comment by ahmed-deeplite** [comment]
Looks good to me, but don’t merge before you add tests.
Downscale the dataset or create a toy. But i would prefer if if you do the tests on the downscaled. Thanks!

**Review by ahmed-deeplite — CHANGES_REQUESTED** [review]
Looks good to me, but we can’t merge with +1% loss in coverage. Thanks!

File: deeplite_torch_zoo/api/datasets/classification/raiseraw.py [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+        shuffle=False,
+    )
+
+    return {"train": train_loader, "test": test_loader}
+
+
+class RAISEDataset(Dataset):
+    def __init__(self,
+                 root,
+                 metadata_file,
```
Comment (ahmed-deeplite) [inline deeplite_torch_zoo/api/datasets/classification/raiseraw.py:66]: It is better if you add default name to the `metadata_file` and use `os.join` with `root` to construct the full path instead of having to provide full path for both root and metadata_file
